airflow/contrib/operators/gcp_bigtable_operator.py

```diff
@@ -0,0 +1,379 @@
+            # the local ones are kept in `gc_rule` property of remote ones
+            if table_column_families[key].gc_rule != self.column_families[key]:
+                self.log.error("Column Family '{}' differs for Table '{}'.".format(key, self.table_id))
+                return False
+        return True
+
+    def execute(self, context):
+        self.instance = self.hook.get_instance(self.project_id, self.instance_id)
+        if not self.instance:
+            raise AirflowException("The Instance '{}' does not exist in project '{}'.".format(
+                self.instance_id, self.project_id))
+        try:
+            self.hook.create_table(
+                self.instance,
+                self.table_id,
+                self.initial_split_keys,
+                self.column_families
+            )
+        except google.api_core.exceptions.AlreadyExists:
+            if not self._compare_column_families():
+                raise AirflowException("Table '{}' already exists with different Column Families.".format(self.table_id))
+            self.log.info('The table already exists')
+
+
+class BigTableTableDeleteOperator(BaseOperator, BigTableValidationMixin):
+    """
+    Deletes the BigTable Table.
+
+    For more details about deleting table have a look at the reference:
+    https://googleapis.github.io/google-cloud-python/latest/bigtable/table.html#google.cloud.bigtable.table.Table.delete
+
+    :type project_id: str
+    :param project_id: The Google Developers Console [project ID or project number]
+    :type instance_id: str
+    :param instance_id: The ID of the Instance the Table belongs
+    :type table_id: str
+    :param table_id: The ID of the Table to be deleted
+    """
+    REQUIRED_ATTRIBUTES = ('project_id', 'instance_id', 'table_id')
+    template_fields = ['project_id', 'instance_id', 'table_id']
+
+    @apply_defaults
+    def __init__(self,
+                 project_id,
+                 instance_id,
+                 table_id,
+                 app_profile_id=None,
+                 *args, **kwargs):
+        self.project_id = project_id
+        self.instance_id = instance_id
+        self.table_id = table_id
+        self.app_profile_id = app_profile_id
+        self._validate_inputs()
+        self.hook = BigTableHook()
+        super(BigTableTableDeleteOperator, self).__init__(*args, **kwargs)
+
+    def execute(self, context):
+        instance = self.hook.get_instance(self.project_id, self.instance_id)
+        if not instance:
+            raise AirflowException("Instance '{}' does not exists.".format(self.instance_id))
+
+        try:
+            self.hook.delete_table(
+                self.project_id,
+                self.instance_id,
+                self.table_id,
+            )
+        except google.api_core.exceptions.NotFound:
+            # It's OK if table doesn't exists
+            self.log.info("The Table '{}' does not exists already.".format(self.table_id))
```

Comment:
Such formatting of messages causes that a string object is created, which then may not be used anywhere, when the login level will be too low.
Formatting parameters should be passed as arguments to the `info` method. 
Example: 
````
self.log.info("The Table '%s' does not exists already.", self.table_id)
````

Reply:
done